fusor-ember-cli/app/templates/satellite.hbs

```diff
@@ -5,7 +5,7 @@
       {{/link-to}}
 
       {{#link-to lifecycleEnvironmentTabRouteName tagName='li' disabled=disableTabLifecycleEnvironment}}
-        <a>1B. Configure Lifecycle Environment</a>
+        <a>1B. Update Availability</a>
```

Comment:
I wasn't aware of this change. I only realized this renaming in the trello card for the radio boxes.